src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -144,7 +144,9 @@ void SubgraphBaseTest::query_model() {
     for (auto&& res : queryNetworkResult) {
         actual.insert(res.first);
     }
-    ASSERT_EQ(expected, actual);
```

Comment:
Why remove test assert?

Reply:
Assert is not applicable for report. We should to catch exceptions.